pandas/stats/moments.py

```diff
@@ -208,6 +208,7 @@ def ensure_compat(dispatch, name, arg, func_kw=None, *args, **kwargs):
         if value is not None:
             kwds[k] = value
 
+    kwargs.pop('freq', None)  # freq removed in 0.22
```

Comment:
this shouldn't be necessary

Reply:
A bunch of tests in ``pandas\tests\test_window.py`` fail without this. 

As I'll clean up that module after finishing up ``how`` (incl. all reference to ``freq`` for ``pd.stats.*``), I suggest leaving it as it is. It not worth it to make the tests pass without this for a module that will be deleted shortly.

Reply:
that's fine, just add a TODO then

Reply:
Well, the whole ``moments.py`` module will be deleted, incl. this small change....

Reply:
I understand, and until that time, pls add a TODO